_data/projects/game-iverse.yml

```diff
@@ -0,0 +1,9 @@
+name: Game-iverse
+desc: A game library that supplies building blocks for games. (New project)
+site: https://github.com/wolffles/game-iverse
+tags:
+- javascript
+- canvas HTML5
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/wolffles/game-iverse/issues
```

Comment:
@wolffles Please change this to https://github.com/wolffles/game-iverse/labels/up-for-grabs

Reply:
I added a file via "create file" on up-for-grab's repo and clicked the pull request button there. I can't figure out how to access this file that I created on spot. Do i have to redo the request or can i just edit it somehow?